src/v/config/configuration.cc

```diff
@@ -4517,6 +4517,13 @@ configuration::configuration()
       "The maximum age of an epoch that the epoch service will publish",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       10min)
+  , epoch_service_cached_epoch_age_soft_limit(
+      *this,
+      "epoch_service_cached_epoch_age_soft_limit",
+      "When the age of a cached epoch reaches the soft limit it triggers a "
+      "background refresh",
```

Comment:
I understand this better as "the local cache duration of a cluster wide epoch".

Reply:
yeh, yours is more direct and literal. fixed